adanet-co2-monitor.ino

```diff
@@ -271,7 +281,8 @@ void setup()
   const int16_t footerY = display.height() - 1 - FOOTER_SIZE * CHAR_HEIGHT;
   if (error == ERROR_NONE)
   {
-    printfAligned(HEADER_SIZE, ALIGN_LEFT, headerY, EPD_BLACK, "% 3.1f%cC", temperature, 0xF7);
+    float disp_temp = (temp_units == 'C') ? temperature : (temperature / 5.f * 9.f + 32.f);
+    printfAligned(HEADER_SIZE, ALIGN_LEFT, headerY, EPD_BLACK, "% 3.1f%c%c", disp_temp, 0xF7, temp_units);
     printfAligned(HEADER_SIZE, ALIGN_RIGHT, headerY, EPD_BLACK, "%3.1f%%", humidity);
```

Comment:
I guess the `%3.1` here is wrong too (should be `%5.1`, in case of 100% humidity)

Reply:
%4.1? Humidity can never be negative.

Reply:
But it could be 100.0% in theory?

Reply:
Right, of course, it should be %5.1